rendezvous/client.go

```diff
@@ -422,7 +423,7 @@ func (c *Client) Close(ctx context.Context, mood Mood) error {
 
 	defer func() {
 		if c.wsClient != nil {
-			c.wsClient.Close()
+			c.wsClient.Close(websocket.StatusNormalClosure, "")
```

Comment:
@vu3rdd can you elaborate on what you're passing here and how it's different from the other lib?

Reply:
The new websocket library has a [different `Close()` interface](https://pkg.go.dev/nhooyr.io/websocket#Conn.Close), that warranted this change.